core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-answer-group-modal.controller.spec.ts

```diff
@@ -141,7 +142,7 @@ describe('Add Answer Group Modal Controller', function() {
     expect($scope.isSelfLoopWithNoFeedback(outcome2)).toBe(false);
   });
 
-  it('should save response', function() {
+  it('should close modal when saving answer group response', function() {
```

Comment:
Done.

Reply:
Hm, I'm confused. I don't think we should change it because I'm asserting when `$uibModal.close` is called, not when the answer group is saved. Here and elsewhere.